### ISSUE TITLE
Proxy support for elm

### ISSUE THREAD

**Issue by cab404** [elm/compiler#2034]
**Quick Summary:** 
With carpet bans from RKN, Russian developers are left with no way to get to https://package.elm-lang.org easily, apart from transparent proxy. 

It would be nice to have socks and http proxy options, so it wouldn't be such a pain to use.

**Comment by cab404** [comment]
Didn't find all the previous issues :\

**Comment by rlefevre** [comment]
https://package.elm-lang.org IP address has been changed a few days ago to unblock it, is it still blocked for you?

Also just for information, elm supports `https` proxy, see for example:
https://gist.github.com/rlefevre/7c21513c3cc14ea23524223ab35e4ca6

**Comment by cab404** [comment]
> https://package.elm-lang.org IP address has been changed a few days ago to unblock it, is it still blocked for you?

Oh wow, it is not blocked. Thank you for the news)

> Also just for information, elm supports https proxy (but not socks)

Yeah, found about the same answer in closed issues. But French proxy is more than useful in case of future bans. Thank you again!
